packages/reactstrap-validation-select/src/AvSelectField.js

```diff
@@ -43,7 +43,7 @@ class AvSelectField extends Component {
     let feedback = null;
     if (validation.errorMessage) {
       const feedbackClasses = classNames('d-block', feedbackClass);
-      feedback = <AvFeedback className={feedbackClasses}>{validation.errorMessage}</AvFeedback>;
+      feedback = <AvFeedback className={feedbackClasses} id={`${id}-error`}>{validation.errorMessage}</AvFeedback>;
```

Comment:
Do you mind changing the suffix to be `-feedback` instead of `-error`. That way it will match the other library we have
https://github.com/Availity/availity-react/blob/master/packages/form/src/Feedback.js#L7

Reply:
@jordan-a-young , I wont. Changing it now

Reply:
I'm also not sure adding the id will help adding error information to this component since react-select's aria-describedby points to the placeholder and screen reader text.

Reply:
@LauRoxx , I'm not following you. I though the id added for the feedback can be referenced with in the AvSelectField by aria-describedby

Reply:
@LauRoxx  and @jordan-a-young , please review the new addition/changes